app/components/find/courses/about_schools_component/view.rb

```diff
@@ -39,17 +40,17 @@ def show_scitt_guidance?
         end
 
         def placements_url
-          if course.has_unpublished_changes? || (course.is_published? && course.is_running?)
-            URI.join(
-              Settings.search_ui.base_url,
-              find_placements_path(course.provider_code, course.course_code)
-            ).to_s
-          else
+          if preview
```

Comment:
Discovered a small edge case where you could bypass the old check. Using this `preview` should fix it